src/psychrometrics/p_sat.js

```diff
@@ -3,6 +3,9 @@ import { round } from "../utilities/utilities.js";
 /**
  * Calculates vapour pressure of water at different temperatures
  *
+ * @public
+ * @memberof psychrometrics
+ *
```

Comment:
psychrometrics don't have a title in the Python version, so if we omit `@docname` I coded it so it will default to the function signature

Reply:
Works for me!